src/groups/mqb/mqbcfg/mqbcfg.xsd

```diff
@@ -271,6 +276,30 @@
     </sequence>
   </complexType>
 
+    <complexType name='TlsConfig'>
+    <annotation>
+      <documentation>
+        certificateAuthority.:
+            A path to the FILE, containing concatenation of known certificates
+            the server can use to reference as its certificate store.
+        certificate..........:
+            A path to the FILE, containing the certificate the broker will use
+            to identify itself to other clients.
+        key..................:
+            A path to the FILE, containing the private key that the broker uses
+            to read the certificate.
+        version..............:
+            Protocol version.
```

Comment:
would be good to give an example of what's expected here e.g. "TLS1.2" 
from the point of view of the server you might want to allow a set of versions or at least version 1.2 or greater. 
It's good to spec out exactly what this parameter means

Reply:
@willhoy we plan to use a list of versions indeed